ocaml/xapi-types/features.ml

```diff
@@ -104,7 +104,7 @@ let keys_of_features =
   ; ( PCI_device_for_auto_update
     , ("restrict_pci_device_for_auto_update", Negative, "PciAU")
     )
-  ; (Xen_motion, ("restrict_xen_motion", Negative, "XenMotion"))
+  ; (Xen_motion, ("restrict_xen_motion", Negative, "Storage_live_migration"))
```

Comment:
This is not _storage_ live migration, but plain live migration, without storage.

Reply:
Will fix